frontend/src/components/report/CSVReportFormatter.ts

```diff
@@ -102,15 +103,19 @@ export class CSVReportFormatter{
     console.log('RESULT HEADERS', result)
     return result
   }
-  private append_metadata_values(index: number){
-    if(!this.values_metadata){
+
+  private append_values(
+      values: any[],
+      index: number){
+
+    if(!values || values.length == 0){
       return null
     }
     let result = ''
-    let row_keys = Object.keys(this.values_metadata[index])
+    let row_keys = Object.keys(values[index])
     for(let i =0;  i < row_keys.length; i++){
```

Comment:
Should we use just a map operator there and do write it more "JavaScript" style?

```typescript
let result = ''
const value_element = values[index]
const row_keys = Object.keys(value_element)

row_keys.map((key:string, key_index: number) => {
  const value = value_element[key]
  result += `${value}`
 
  if (key_index != row_keys.length - 1) result += ','
})
```

Reply:
Thanks Vitalii

I added it here https://github.com/diffgram/diffgram/commit/888b0ecae167b11ddb70d2a369d4bb900a1053a1

Not quite as sure on this style point, but maybe something we can surface next time need to discuss JS style trade offs on call